src/test/scala/com/github/dnvriend/serializer/avro4s/AvroSENoOldClassesTest.scala

```diff
@@ -68,172 +68,47 @@ class AvroSENoOldClassesTest extends AvroTestSpec {
   }
 
   "v1 to v2" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV2]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wf"),
-      oldSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 1,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" }
-        |  ]
-        |}
-      """.stripMargin.toSchema,
-      newSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 2,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" },
-        |   { "name" : "director", "type" : "string", "default" : "unknown" }
```

Comment:
The default avro serializer we use generates a default schema for the case class, I think that it doesn't use a schema version. I still don't know how Avro encodes schema versioning, but I've  read its the first byte 0x0 and then 4 bytes for the version, but when I parse it I get `06666f6f8c`, with:

```scala
 implicit class ArrayOps(xs: Array[Byte]) {
    def encode[OUT](implicit ev: Encoder[Array[Byte], OUT]): OUT = ev.encode(xs)
    def toHex: String = xs.map(byte ⇒ f"$byte%02x").mkString("")
  }
```